src/index.js

```diff
@@ -821,12 +929,16 @@ export default function (babel) {
     path.traverse({
 
       ForInArrayStatement(path) {
+        const node = path.node;
+        const nb = (...args) => nodeBefore(node, ...args);
+        const n = (...args) => nodeAt(node, ...args);
+
         // If array is a complex expression, generate: const _arr = expr
         let refId;
         if (!path.get("array").isIdentifier()) {
           refId = path.scope.generateUidIdentifier("arr");
           path.insertBefore(
-            t.variableDeclaration("const", [ t.variableDeclarator(refId, path.node.array )])
+            nb("variableDeclaration", "const", [ nb("variableDeclarator", refId, path.node.array )])
```

Comment:
You may be right, I won't weigh in yet because I haven't diagnosed it -- I'm kind of going through these things in the order that the test runner iterates the fixtures, so comprehensions might be a while yet.